pandas/core/strings.py

```diff
@@ -1651,17 +1651,34 @@ def str_translate(arr, table, deletechars=None):
 
 def str_get(arr, i):
     """
+    Extract element for each component.
+
     Extract element from lists, tuples, or strings in each element in the
     Series/Index.
 
     Parameters
     ----------
     i : int
-        Integer index (location)
+        Integer index (location).
 
     Returns
     -------
     items : Series/Index of objects
+
+    Examples
+    --------
+    >>> x = pd.Series(["String", (1, 2, 4), ["a", "b", "c"]])
```

Comment:
The standard way is to name the Series `s` and not `x`.

Reply:
I see. I'd change this.